README.md

```diff
@@ -49,36 +48,58 @@ Value returned by query resolver must be a kaminari object or implements its pag
 }
 ```
 
-```
+```json
 {
-  'data': {
-    'checklists': {
-      'collection': [
+  "data": {
+    "checklists": {
+      "collection": [
         {
-          'id': '93938bb3-7a6c-4d35-9961-cbb2d4c9e9ac',
-          'name': 'Apple'
+          "id": "93938bb3-7a6c-4d35-9961-cbb2d4c9e9ac",
+          "name": "Apple"
         },
         {
-          'id': 'b1ee93b2-579a-4107-8454-119bba5afb63',
-          'name': 'Mango'
+          "id": "b1ee93b2-579a-4107-8454-119bba5afb63",
+          "name": "Mango"
         }
       ],
-      'metadata': {
-        'totalPages': 25,
-        'totalCount': 50,
-        'currentPage': 2,
-        'limitValue': 2
+      "metadata": {
+        "totalPages": 25,
+        "totalCount": 50,
+        "currentPage": 2,
+        "limitValue": 2
```

Comment:
I thought the README needed more colors, and then this was not valid JSON so I had to fix it ;-)

Reply:
Weird, thx for fixing that 👍